api/checks/summaries/compile.go

```diff
@@ -114,3 +129,19 @@ func (r Regressed) GetCheckState() CheckState {
 func (r Regressed) GetSummary() (string, error) {
 	return compile(&r, "regressed.md")
 }
+
+// GetActions returns the actions that can be taken by the user.
+func (r Regressed) GetActions() []*github.CheckRunAction {
+	return []*github.CheckRunAction{
+		&github.CheckRunAction{
+			Identifier:  "recompute",
+			Label:       "Recompute",
+			Description: "Recompute against the latest master run",
+		},
+		&github.CheckRunAction{
+			Identifier:  "ignore",
+			Label:       "Ignore",
+			Description: "Mark these results as expected (passing)",
```

Comment:
Not sure I understand the "(passing)" bit means. It will be for failing tests that this button even appears, right?

Reply:
If you don't, others might not either. Yes, these buttons are only shown for a "Regressed" outcome.

It's supposed to imply that the currently-failing check would then be marked as passing.